calibr8/contrib/noise.py

```diff
@@ -37,7 +37,7 @@ def to_pymc(*params):
 
 class LogNormalNoise(DistributionMixin):
     """Log-Normal noise, predicted in logarithmic mean and standard deviation.
-    ⚠ This corresponds to the NumPy/Aesara/PyMC parametrization!
+    ⚠ This corresponds to the NumPy/PyTensor/PyMC parametrization!
```

Comment:
Technically this could still be Aesara since you allow it as a fallback in case PyTensor is not installed.

Reply:
Yes, but I wanted this to not need another change once we eventually drop Aesara support